examples/app.py

```diff
@@ -31,10 +31,11 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
         pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
+        pip install git+git://github.com/inveniosoftware/dojson.git
```

Comment:
Is it possible to make a release of dojson as-is, and then merge this afterwards? It's just that usually these tiny updates after something else is released are forgotten.